src/services/rhsm/__tests__/__snapshots__/rhsmHelpers.test.js.snap

```diff
@@ -0,0 +1,133 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RhsmHelpers billingMetrics should return parsed API responses in a consumable metric format, basic 1`] = `{}`;
+
+exports[`RhsmHelpers billingMetrics should return parsed API responses in a consumable metric format, multiple providers, multiple accounts 1`] = `
+{
+  "dolorApi": {
+    "accounts": [],
+    "firstProvider": undefined,
+    "firstProviderAccount": undefined,
+    "firstProviderNumberAccounts": undefined,
+    "hasUniqueAccounts": false,
+    "hasUniqueProviders": false,
+    "numberProviders": 0,
+    "providers": [],
+  },
+  "loremApi": {
+    "accounts": [
+      {
+        "id": "123",
+        "provider": "ipsum",
+        "type": "loremApi",
+      },
+      {
+        "id": "456",
+        "provider": "lorem",
+        "type": "loremApi",
+      },
+      {
+        "id": "012",
+        "provider": "dolor",
+        "type": "loremApi",
+      },
+    ],
+    "firstProvider": "ipsum",
+    "firstProviderAccount": "123",
+    "firstProviderNumberAccounts": 1,
+    "hasUniqueAccounts": true,
+    "hasUniqueProviders": true,
+    "numberProviders": 3,
+    "providers": [
+      {
+        "id": "123",
+        "provider": "ipsum",
+        "type": "loremApi",
+      },
+      {
+        "id": "012",
+        "provider": "dolor",
+        "type": "loremApi",
+      },
+    ],
+  },
+}
+`;
+
+exports[`RhsmHelpers billingMetrics should return parsed API responses in a consumable metric format, single provider, multiple accounts 1`] = `
+{
+  "dolorApi": {
+    "accounts": [],
+    "firstProvider": undefined,
+    "firstProviderAccount": undefined,
+    "firstProviderNumberAccounts": undefined,
+    "hasUniqueAccounts": false,
+    "hasUniqueProviders": false,
+    "numberProviders": 0,
+    "providers": [],
+  },
+  "loremApi": {
+    "accounts": [
+      {
+        "id": "123",
+        "provider": "lorem",
+        "type": "loremApi",
+      },
+      {
+        "id": "456",
+        "provider": "lorem",
+        "type": "loremApi",
+      },
+      {
+        "id": "012",
+        "provider": "lorem",
+        "type": "loremApi",
+      },
+    ],
+    "firstProvider": "lorem",
+    "firstProviderAccount": "123",
+    "firstProviderNumberAccounts": 3,
+    "hasUniqueAccounts": true,
+    "hasUniqueProviders": false,
+    "numberProviders": 1,
+    "providers": [],
+  },
+}
```

Comment:
required: this isn't technically the scenario we intended to return... it should be a unique provider, note the `hasUniqueProviders` flag

confirm the scenario in test, add an additional scenario where the provider is unique but still falls into the test description

Reply:
added to scenario variations, results and descriptions align to what is intended